packages/contracts/src/dollar/mocks/MockERC4626.sol

```diff
@@ -57,7 +57,7 @@ contract MockERC4626 is ERC4626 {
     {
         require(assets <= maxWithdraw(owner), "ERC4626: withdraw more than max");
 
-        uint256 shares = previewWithdraw(assets);
+        shares = previewWithdraw(assets);
```

Comment:
Seems suspicious that uint256 was removed here. Why?

Reply:
declared in return